lib/cli/src/versions.json

```diff
@@ -1,61 +1,61 @@
 {
-  "@storybook/addon-a11y": "6.2.0-rc.9",
```

Comment:
This change was somehow automated by tooling. Please let me know if I should yank it.